### PR TITLE
Consolidate Sample 

### PR DESCRIPTION
**Pull request by shaishavgandhi05**
**Description**: Consolidate sample to open up different activities. Have also included a brief description before the Button to describe the sample.

### CODE REVIEW

**Comment by hzsweers** [comment]
Would you mind adding a screenshot(s) or screenrecording of the new UI for posterity?

**Comment by shaishavgandhi05** [comment]
Here's a screenshot:

autodispose

Just opens up the Activities. One concern is that both `MainActivity` and `KotlinActivity` don't really display anything. They just log the Observable sequence.

File: sample/src/main/kotlin/com/uber/autodispose/sample/HomeActivity.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ */
+class HomeActivity: AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_home)
+
+    val mainActivityButton: Button = findViewById(R.id.mainActivityButton)
+    mainActivityButton.setOnClickListener {
+      startActivity(MainActivity::class.java)
```
[the remark this answers is not in the document]
Comment (hzsweers) [reply sample/src/main/kotlin/com/uber/autodispose/sample/HomeActivity.kt]: Works for me!

File: sample/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -20,4 +20,7 @@
   <string name="title_activity_main">AutoDispose Demo</string>
   <string name="title_activity_arch_component">AutoDispose ArchComponents Demo</string>
   <string name="load_image">Load Image</string>
+  <string name="main_activity_description">Sample Java Activity which uses Architecture Components lifecycle to manage subscription</string>
```
[the remark this answers is not in the document]
Comment (hzsweers) [reply sample/src/main/res/values/strings.xml]: How about `Sample Java Activity demonstrating integration with Architecture Components Lifecycle`?

**Comment by hzsweers** [comment]
Nit: Can we remove the "AutoDispose" prefix in the button texts in favor of just a toolbar with `AutoDispose Demo` as the title?

**Comment by hzsweers** [comment]
thanks!

**Pull request merged**